ui/app/src/state/actions/dialogs.ts

```diff
@@ -303,3 +304,25 @@ export const rtePickerActionResult = /*#__PURE__*/ createAction<{ path: string;
   'RTE_PICKER_ACTION_RESULT'
 );
 // endregion
+
+// region BrokenReferences Cancellation
+
+export const showBrokenReferencesDialog = /*#__PURE__*/ createAction<Partial<BrokenReferencesDialogStateProps>>(
+  'SHOW_BROKEN_REFERENCES_DIALOG'
+);
+
+export const closeBrokenReferencesDialog = /*#__PURE__*/ createAction<StandardAction>('CLOSE_BROKEN_REFERENCES_DIALOG');
+
+export const brokenReferencesDialogClosed = /*#__PURE__*/ createAction<StandardAction>(
+  'BROKEN_REFERENCES_DIALOG_CLOSED'
+);
+
+export const fetchBrokenReferences = /*#__PURE__*/ createAction<StandardAction>('FETCH_BROKEN_REFERENCES');
```

Comment:
1. The request was to define the **PAYLOADS**
2. createAction receives the type for the payload, this makes no sense